internal/storage/rollup_provider.go

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: 2023 PK Lab AG <contact@pklab.io>
+// SPDX-License-Identifier: MIT
+
+package storage
+
+import (
+	"github.com/dipdup-net/indexer-sdk/pkg/storage"
+	"github.com/uptrace/bun"
+)
+
+//go:generate mockgen -source=$GOFILE -destination=mock/$GOFILE -package=mock -typed
+type IRollupProvider interface {
+	storage.Table[*RollupProvider]
+}
+
+// RollupProvider -
+type RollupProvider struct {
```

Comment:
it looks like we have two different ways to register rollup providers:

- by address only (this is a more general and flexible way, since we do not need to register all namespaces)
- by namespace and address pair // like this code